lua/wire/client/e2descriptions.lua

```diff
@@ -54,6 +54,7 @@ E2Helper.Descriptions["sqrt(n)"] = "Returns the Square Root of the Argument"
 E2Helper.Descriptions["cbrt(n)"] = "Returns the Cube Root of the Argument"
 E2Helper.Descriptions["root(nn)"] = "Returns the Nth Root of the first Argument"
 E2Helper.Descriptions["e()"] = "Returns Euler's Constant"
+E2Helper.Descriptions["frexp(n)"] = "Returns a mantissa and exponent of given floating-point number as vector2 type (x is mantissa, y is exponent)"
```

Comment:
I feel like this would be better read

`Returns the mantissa and exponent of a given floating-point number as a vec2 (x is mantissa, y is exponent)`